lib/SPIRV/SPIRVToOCL.h

```diff
@@ -215,6 +215,11 @@ class SPIRVToOCLBase : public InstVisitor<SPIRVToOCLBase> {
   /// - OCL1.2: barrier
   virtual void visitCallSPIRVControlBarrier(CallInst *CI) = 0;
 
+  /// Transform split __spirv_ControlBarrier barrier to:
```

Comment:
Guess, it should be __spirv_ControlBarrierArriveINTEL and __spirv_ControlBarrierWaitINTEL

Reply:
Yes, I've fixed these comments so now they are all a variant of:

```c
  /// Transform split __spirv_ControlBarrierArriveINTEL and
  /// __spirv_ControlBarrierWaitINTEL barrier to...
```